_posts/2023-01-04-trino-2022-the-rabbit-reflects.md

```diff
@@ -0,0 +1,581 @@
+    * Improved latency when querying Iceberg tables with many files. (401)
+    * Support for `UPDATE` in Iceberg connector. (381)
+    * Allow changing file and table format with the Iceberg connector. (380)
+    * File-based access control for the Iceberg connector. (367/368)
+* Although it is seeming more and more like Hive is already on its way out,
+ there are many that still depend on the Hive connector to be performant and
+ adding new features when it makes sense. S3 Select pushdown for JSON data in Hive connector.
+    * Support for IBM Cloud Object Storage in Hive. (401)
+    * Rename properties controlling Hive view parsing. (380)
+    * Improved performance when querying partitioned Hive tables. (373)
+    * `flush_metadata_cache()` procedure for the Hive connector. (367/368)
+* Improved performance for Parquet data in Delta Lake, Hive and Iceberg connectors. (398)
+* Faster writing of Parquet data in Iceberg and Delta Lake. (391)
+* Support for Amazon S3 Select pushdown for JSON files. (392)
+* Support access to S3 via HTTP proxy. (373)
+
+### Other connectors
+
+A major feature of Trino is the availability of other connectors to query all
+sorts of databases with SQL. All with the speed, and federation from Trino.
+Following as some of the major improvements that landed for these connectors in
+2022:
+
+* New MariaDB connector
+* Performance improvements with various pushdowns in the MongoDB, MySQL, Oracle,
+ PostgreSQL and SQL Server connectors.
+* Support for bulk data insertion in SQL Server connector.
+* Added a query passthrough table function to numerous connectors.
+* Expanded SQL features for various connectors by adding support for
+ `TRUNCATE TABLE`, `DELETE`, `CREATE`/`DROP` `SCHEMA`, `INSERT`, and others.
+* Update Cassandra connector to support v5 and v6 protocols.
+* A collection of improvements on the Pinot and BigQuery connectors
+
+### Bug fixes
+
+Any software includes issues and bugs, Trino included. Thanks to our big user
+community we learned about many of them, and fixed even more. Continue to test
+new releases and report issues. Check out[ all the release notes for details
+]({{site.url}}/docs/current/release.html#releases-2022).
+
+## Updates in the Trino ecosystem community
+
+Outside of the excitement within the main Trino project, there was a great deal
+going on in the larger Trino community and ecosystem:
+
+### Trino: The Definitive Guide second edition
+
+Martin, Manfred, and Matt released the [second version of the Trino Definitive
+Guide]({% post_url 2022-10-03-the-definitive-guide-2 %}). This 
+version fixed errata, added the deployment process to include newer Kubernetes
+installation methods, and updated features with the latest version of Trino that
+was not available in the initial version of the book. Along with this, there are
+[efforts underway to translate this book](https://simpligility.ca/2022/12/trino-guide-for-everyone-in-2023/)
```

Comment:
we either remove it here .. or in the section about regions earlier

Reply:
Why though? It's such a long blog post that they may find skip one section and only see the other. I think it's relevant to both.

Reply:
I can remove the link on the one above, but I think that we should mention it in both places.